newIDE/app/src/MainFrame/index.js

```diff
@@ -1674,6 +1678,10 @@ class MainFrame extends React.Component<Props, State> {
       snackMessageOpen: false,
     });
 
+  quitApp = () => {
+    electron.app.quit()
```

Comment:
Let's check if electron is not null :) If we're on the web-app, it will be null - and while it does not make sense to have this on the web-app, it's a good idea to always use defensive programming.